packages/opentelemetry-metrics/src/BasicMeterRegistry.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConsoleLogger } from '@opentelemetry/core';
+import * as types from '@opentelemetry/types';
+import { Logger } from '@opentelemetry/types';
+import { Meter } from '.';
+import { DEFAULT_CONFIG, MeterConfig } from './types';
+
+/**
+ * This class represents a basic tracer registry which platform libraries can extend
+ */
+export class BasicMeterRegistry implements types.MeterRegistry {
```

Comment:
I would prefer to keep `MeterRegistry` instead of `BasicMeterRegistry`, I don;t expect to see `NodeMeterRegistry`.